examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -384,18 +384,40 @@ export class TaskList extends DataObject implements ITaskList {
 	 * DataObject, by registering an event listener for changes to the task list.
 	 */
 	protected async hasInitialized(): Promise<void> {
-		const saved = this.root.get<IFluidHandle<SharedMap>>("savedData");
+		const [saved, draft] = await Promise.all([
+			this.root.get<IFluidHandle<SharedMap>>("savedData"),
+			this.root.get<IFluidHandle<SharedMap>>("draftData"),
+		]);
```

Comment:
Why are we using Promise here as against just doing a simple get?

Reply:
I added this as a small optimization so that we can call both `get` commands at the same time for saved and draft.

Reply:
The calls to `this.root.get<>()` don't return a Promise though, so this doesn't have any effect. The two calls will run in order and synchronously; the first one will return before the second one can start executing.

Further down (when calling `await saved.get()` and `await draft.get()`) we could potentially use this pattern for a bit of potential perf gain, since those two lines are acting on Promises, but in an example app I'd lean towards keeping the current approach which is easier to read and reason about. We don't need every last ounce of performance here.